### ISSUE TITLE
`Unknown restclient hook type jq-set-var` With Latest Version

### ISSUE THREAD

**Issue by tompurl** [pashky/restclient.el#282]
I'm using Doom emacs. The pinned version is a few commits behind so I added the following to my `packages.el` file:

``` emacs-lisp
(package! restclient :pin "ae79e7dd283890072da69b8f48aeec1afd0d9442" )
```

I then ran `doom sync -u` and confirmed that the newer version was installed. 

I also have `jq-mode` installed here:

- `~/.emacs.d/./.local/straight/repos/jq-mode`

However, when I attempt the use `jq-set-var` I get the following message:

```
Unknown restclient hook type jq-set-var
```

I have also tried running the following commands from a scratch buffer before executing a restclient query:

```
(require 'restclient)
(require 'jq-mode)
```

**Comment by shackra** [comment]
yeah, I'm experiencing this as well having jq installed and using the commit ae79e7dd2838

**Comment by shackra** [comment]
Okay, after toying around with this issue, I got a workaround for Doom Emacs users:

```
(use-package! restclient
  :defer)
(after! restclient
  (require 'restclient-jq))
```

yes, we need to manually `require` the `restclient-jq` module.

And that's it, basically.

**Comment by tompurl** [comment]
This is excellent, thank you!

**Comment by micahbf** [comment]
If anyone else comes across this, doom now has this built in.

In `init.el`, you just need to enable the `jq` flag for the `rest` module, i.e. `(rest +jq)`
